src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -6,28 +6,33 @@ import { tokens, tokensToPin } from '$lib/derived/tokens.derived';
 import { balancesStore } from '$lib/stores/balances.store';
 import type { Token, TokenUi } from '$lib/types/token';
 import { filterTokensForSelectedNetwork } from '$lib/utils/network.utils';
-import { pinTokensWithBalanceAtTop, sortTokens } from '$lib/utils/tokens.utils';
+import {
+	filterEnabledTokens,
+	pinTokensWithBalanceAtTop,
+	sortTokens
+} from '$lib/utils/tokens.utils';
 import { derived, type Readable } from 'svelte/store';
 
+/**
+ * All enabled by user tokens.
+ */
+const enabledTokens: Readable<Token[]> = derived([tokens], filterEnabledTokens);
+
 /**
  * All tokens matching the selected network or chain fusion, regardless if they are enabled by the user or not.
  */
-const networkTokens: Readable<Token[]> = derived(
-	[tokens, selectedNetwork, pseudoNetworkChainFusion],
+const enabledNetworkTokens: Readable<Token[]> = derived(
+	[enabledTokens, selectedNetwork, pseudoNetworkChainFusion],
 	filterTokensForSelectedNetwork
 );
 
-const enabledNetworkTokens: Readable<Token[]> = derived([networkTokens], ([$networkTokens]) =>
-	$networkTokens.filter((token) => ('enabled' in token ? token.enabled : true))
-);
-
 /**
- * It isn't performant to post filter again the Erc20 tokens that are enabled for the specific selected network or no network selected but, it's code wise convenient to avoid duplication of logic.
+ * It isn't performant to post filter again the Erc20 tokens that are enabled but, it's code wise convenient to avoid duplication of logic.
  */
 export const enabledErc20NetworkTokens: Readable<Erc20Token[]> = derived(
-	[enabledNetworkTokens],
-	([$enabledNetworkTokens]) =>
-		$enabledNetworkTokens.filter(({ standard }) => standard === 'erc20') as Erc20Token[]
+	[enabledTokens],
```

Comment:
Is this correct and expected? Shouldn't this be renamed and shouldn't it be moved to the `tokens.derived` module?

Reply:
The change is expected - this derived store is used to load balances in `LoaderBalances`, and the idea is that now we fetch all enabled tokens balances (as if Chain Fusion were selected in `NetworksSwitcher`).

And yes, It makes sense to move it to `tokens.derived`. If that's okay, I would do it in a separate PR, as I would also move the IC tokens store, as well as rename both: `enabledErc20NetworkTokens` -> `enabledErc20Tokens`, `enabledIcNetworkTokens` -> `enabledIcTokens`.